store/store.go

```diff
@@ -1143,10 +1144,10 @@ func (s *Store) Download(name string, downloadInfo *snap.DownloadInfo, pbar prog
 	return w.Name(), w.Sync()
 }
 
+var downloadBackoffs = []time.Duration{113, 191, 331, 557, 929, 0} // 3 pₙ₊₁ ≥ 5 pₙ; last entry should be 0
```

Comment:
also why time.Duration if we multiply by milliseconds below,  also as durations these ones would be silly short